relatio/_embeddings.py

```diff
@@ -89,6 +89,9 @@ def __init__(
             self._sif_dict = {}
             self._use_sif = False
 
+        # to determine the size of vectors
+        self.size_vectors = self._get_default_vector("the").shape[0]
```

Comment:
This is a hack and we should try to do it differently. One big drawback is that "the" is language specific.
I suggest to add `size_vectors` as a property of `EmbeddingsBase` and we try to get it as clean as possible for each derived class.

`Embeddings` class will just use it.

Reply:
Agreed

Reply:
What about making a list of supported models with the language and the size of the vectors in a separate file?

Reply:
I added the supported models as a dict with language and size_vectors.

Reply:
We concluded to keep it as it is and add a TODO for future.